seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/xa/XaFacadeImplAutoLoad.java

```diff
@@ -89,18 +92,16 @@ public void open() throws SQLException {
         XADataSource ds;
         try {
             ds = (XADataSource) DataSourceUtils.buildCommonDataSource(jdbcConnectionOptions);
-        }
-        catch (Exception e) {
-            throw new SQLException(e);
+        } catch (Exception e) {
+            throw new JdbcConnectorException(JdbcConnectorErrorCode.CONNECT_DATABASE_FAILED, e);
         }
         xaConnection = ds.getXAConnection();
         xaResource = xaConnection.getXAResource();
         if (jdbcConnectionOptions.getTransactionTimeoutSec().isPresent()) {
             try {
                 xaResource.setTransactionTimeout(jdbcConnectionOptions.getTransactionTimeoutSec().get());
-            }
-            catch (XAException e) {
-                throw new SQLException(e);
+            } catch (XAException e) {
+                throw new JdbcConnectorException(JdbcConnectorErrorCode.XA_EXCEPTION, e);
```

Comment:
add info "unable to set XA transaction timeout"

Reply:
ok